controllers/fio/job.go

```diff
@@ -33,14 +33,14 @@ func NewJob(cr *perfv1alpha1.Fio) *batchv1.Job {
 		Namespace: cr.Namespace,
 	}
 
-	fioCmdLineArgs := []string{}
-	fioCmdLineArgs = append(fioCmdLineArgs,
-		qsplit.ToStrings([]byte(cr.Spec.CmdLineArgs))...)
-	fioCmdLineArgs = append(fioCmdLineArgs, cr.Spec.BuiltinJobFiles...)
+	fioArgs := []string{}
+	fioArgs = append(fioArgs,
```

Comment:
https://github.com/xridge/kubestone/pull/128#discussion_r321097107

😜

Reply:
Addressed in all stones.